stdlib/_operator.pyi

```diff
@@ -1,4 +1,5 @@
 import sys
+from builtins import _GetItemIterable
```

Comment:
Could you use `_typeshed.SupportsGetItem` instead?

Reply:
I cannot as-is, because `SupportsGetItem` defines `__contains__`, AND it is defined as `def __contains__(self, __x: object) -> bool: ...`.
Not all indexable types (including buffer-like ones) define `__contains__` \*

bytes defines it as `def __contains__(self, __o: SupportsIndex | bytes) -> bool: ...`, and I am uncertain of the change required to make it work. (other than just typing the parameter `__x` as `Any`).

\* On that note, `mmap` missing `__contains__` seems to be a mistake:
```py
>>> from mmap import mmap
>>> 0 in mmap(0, 5)     
False
>>> b'\x00' in mmap(0, 5)
True
```